sequins.go

```diff
@@ -375,6 +377,11 @@ func (s *sequins) refreshAll() {
 }
 
 func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == "/fetch" && strings.HasPrefix(r.RemoteAddr, localhost) && r.Method == "POST" {
```

Comment:
What happens if a db is named `fetch` Maybe we should update the docs to reflect that fetch and healthz are reserved db names.

Reply:
It has to be a post request to trigger the request, so I think this is safe.